src/try-convert/Program.cs

```diff
@@ -25,21 +26,24 @@ private static async Task<int> Main(string[] args)
                 .AddOption(new Option(new[] { "-p", "--project" }, "The path to a project to convert", new Argument<string>()))
                 .AddOption(new Option(new[] { "-w", "--workspace" }, "The solution or project file to operate on. If a project is not specified, the command will search the current directory for one.", new Argument<string>()))
                 .AddOption(new Option(new[] { "-m", "--msbuild-path" }, "The path to an MSBuild.exe, if you prefer to use that", new Argument<string>()))
+                .AddOption(new Option(new[] { "-tfm", "--target-framework" }, "The name of the framework you would like to upgrade to", new Argument<string>()))
+                .AddOption(new Option(new[] { "--preview" }, "Use preview SDKs as part of conversion", new Argument<string>()))
                 .AddOption(new Option(new[] { "--diff-only" }, "Produces a diff of the project to convert; no conversion is done", new Argument<bool>()))
                 .AddOption(new Option(new[] { "--no-backup" }, "Converts projects and does not create a backup of the originals.", new Argument<bool>()))
                 .Build();
 
             return await parser.InvokeAsync(args.Length > 0 ? args : new string[] { "-h" }).ConfigureAwait(false);
         }
 
-        public static int Run(string project, string workspace, string msbuildPath, bool diffOnly, bool noBackup)
+        public static int Run(string project, string workspace, string msbuildPath, string tfm, bool? allowPreviews, bool diffOnly, bool noBackup)
```

Comment:
I don't think `allowPreviews` has to be a `bool?`, note that there are other bools and if the value is unspecified it's just `false`.